springboot/.idea/springboot.iml

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module org.jetbrains.idea.maven.project.MavenProjectsManager.isMavenModule="true" type="JAVA_MODULE" version="4">
```

Comment:
Ignore iml files. They're IntilliJ-specific project configs.
Also, ignore the entire `.idea/` directory.

Other than that just keep an eye out for excessive spacing.

Reply:
Noted!